plugins/webhooks/index.js

```diff
@@ -595,10 +604,7 @@ exports.register = (server, options, next) => {
                     request.log(['webhook', hookId], `Received event type ${type}`);
 
                     if (/\[(skip ci|ci skip)\]/.test(parsed.lastCommitMessage)) {
-                        message = 'Skipping due to the commit message';
-                        request.log(['webhook', hookId], message);
```

Comment:
are we getting rid of this log statement?

Reply:
hmm since we are not skipping this webhook path anymore, we should skip it right?

Reply:
oh I see
I thought the message would be lost
nvm!